app/models/service.rb

```diff
@@ -26,6 +26,9 @@ class Service < ApplicationRecord
   after_create :create_default_metrics, :create_default_service_plan, :create_default_proxy
   after_commit :update_notification_settings
 
+  after_create :create_and_publish_service_created_event
```

Comment:
I dunno why, but this doesn't work: `after_create :create_and_publish_service_created_event`
Anyway it is not a problem right now because out problem was not for creating but for deleting, and the next line is working 😄

Reply:
`after_commit` requires `disable_transactional_fixtures!` in the test to pass 😉